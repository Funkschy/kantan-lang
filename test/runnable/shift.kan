import "io";

def main() {
    let i: i32 = 'a' << 2;
    io.printf("%d\n", i);
    io.printf("%d\n", i >> 1);

    io.printf("%d\n", 1 << 3);
}
