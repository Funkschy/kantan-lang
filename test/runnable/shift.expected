388
194
8
